common/log.go

```diff
@@ -1,11 +1,11 @@
 package common
 
 import (
-	"os"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
+	"os"
```

Comment:
Why do @lm2343635 you change this order?
Auto-format?

Reply:
Auto-format